### PR TITLE
OM-44 - Create initial UI

### PR DESCRIPTION
**Pull request by jimppa**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "extends": ["react-app", "plugin:@typescript-eslint/recommended"],
+  "overrides": [
+    {
+      "files": ["**/*.tsx"],
```
Comment (midori0507) [inline .eslintrc.json]:
```suggestion
      "files": ["**/*.tsx", "**/*.ts"],
```

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```
Comment (midori0507) [inline .gitignore]: Without `/` maybe?

File: src/i18n/i18n.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import i18n from 'i18next';
```
Comment (midori0507) [inline src/i18n/i18n.ts]: rename to i18nInit?

Comment (jimppa) [reply src/i18n/i18n.ts]: I also first thought about init. However the default export from this file `i18n`-instance/promise is passed to the provider in BrowserApp. What do you think?

File: .gitignore (hunk 2 above)
Comment (jimppa) [reply .gitignore]: This is what CRA does by default. What is the difference, I have no idea?

Comment (midori0507) [reply .gitignore]: Ah so nvm then!

File: src/i18n/i18n.ts (hunk 3 above)
Comment (midori0507) [reply src/i18n/i18n.ts]: You can name them as anything when you import it, default export is flexible
`import i18n from ./i18nInit.ts`

**Pull request merged**